xLights/SubModelsDialog.h

```diff
@@ -316,6 +316,7 @@ class SubModelsDialog : public wxDialog
     void OnInit(wxInitDialogEvent& event);
     void OnNodesGridCellRightClick(wxGridEvent& event);
     void OnCheckBox_OutputToLightsClick(wxCommandEvent& event);
+    void OnCancel(wxCloseEvent& event);
     //*)
```

Comment:
This change appears inside the wxSmith block and I don't see a change to the .wxs file?